dpbench/benchmarks/black_scholes/black_scholes_numba_dppy_k.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache 2.0
+
+
+from math import log, sqrt, exp, erf
+import numba_dppy as nb
```

Comment:
Are you using an older version and not dpex from dppy/label/dev? Let us not do so, but instead fix whatever is filing in dpex and get it to work. Otherwise, we will have to change this